Jenkinsfile

```diff
@@ -10,18 +8,16 @@ elifePipeline {
 
         stage 'Build images', {
             checkout scm
-            dockerComposeBuild(commit)
-        }
-
-        stage 'Check version', {
-            candidateVersion = dockerComposeRunAndCaptureOutput(
-                "sciencebeam",
-                "./print_version.sh",
-                commit
-            ).trim()
-            echo "Candidate version: ${candidateVersion}"
-            isNew = sh(script: "git tag | grep v${candidateVersion}", returnStatus: true) != 0
-            echo "isNew: ${isNew}"
+            // TODO: extract into dockerComposeBuild
```

Comment:
Are we waiting for extracting this before it can go in?

Reply:
not a blocker, shouldn't hold this

Reply:
Okay, any other objections against merging it now?